venvs/common.py

```diff
@@ -8,9 +8,25 @@
 import attr
 import click
 import filesystems.native
+import pytoml
+import tomlkit
 import virtualenv as virtualenv_for_path
 
 
+def load_config(filesystem, locator):
+    with filesystem.open(locator.root.descendant("virtualenvs.toml")) as venvs:
+        text = venvs.read()
+        try:
+            return tomlkit.loads(text)
+        except ValueError:
```

Comment:
What cases would this raise a `ValueError` but succeed with `pytoml`?

And in those cases, looks like you've lost the behavior to preserve order when loading via pytoml.